Moulika/ass3.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Register</h1>
+    <form id="Register">
+        <label for="First name">First Name</label>
+        <input type="text" placeholder="First Name" required autocomplete="given-name">
+        <label for="Last name">Last Name</label>
+        <input type="text" placeholder="Last Name"required autocomplete="family-name">

```

Comment:
Incorrect usage of `label`. `for` attribute corresponds to the `id`. Refer this: [HTML \<label> Tag](https://www.w3schools.com/tags/tag_label.asp)

Reply:
Still incomplete.